go.mod

```diff
@@ -12,13 +12,17 @@ require (
 	github.com/lib/pq v1.10.9
 	github.com/sethvargo/go-envconfig v1.0.1
 	go.uber.org/zap v1.27.0
-	golang.org/x/crypto v0.22.0 // indirect
+	golang.org/x/crypto v0.22.0
 )
 
 require (
 	github.com/aws/aws-sdk-go v1.53.9
+<<<<<<< HEAD
```

Comment:
solve this conflict. 
paling gampang remove2 <<<<<<< HEAD  && =======
 && >>>>>>> d6eb7f016daf345022fe9e1a5536874dd1b2613d
 trus run go mod tidy